apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -415,7 +415,7 @@ const EventTypePage = (props: EventTypeSetupProps) => {
         !leaveWithoutAssigningHosts.current &&
         !!team &&
         (hosts.length === 0 || assignedUsers.length === 0) &&
-        (url === "/event-types" || paths[1] !== "event-types")
+        (url === "/event-types" || (url !== "/event-types?" && paths[1] !== "event-types"))
```

Comment:
We are trying to differentiate b/w "navigating to `/event-types` when the event-type is deleted" and "a regular navigation to the page"

Instead of indirectly tackling this by adding `?` we can directly use the delete status